concent_api/core/models.py

```diff
@@ -456,7 +472,19 @@ def response_type_enum(self):
     delivered            = BooleanField(default = False)
 
     subtask              = ForeignKey(Subtask, blank = True, null = True)
-    created_at           = DateTimeField(default = timezone.now)
+
+    created_at = DateTimeField(default=timezone.now)
+
+    modified_at = DateTimeField(default=timezone.now)
+
+    # Overwritten method to save object with modified timestamp
+    # created_at field will be fulfilled only when object will be created
+    # modified_at field will be overwritten every time when object will be modified
+    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
+        if not self.id:
+            self.created_at = timezone.now()
+        self.modified_at = timezone.now()
```

Comment:
For `modified_at` fields please use `auto_add=True`.

Reply:
Done